src/lib.rs

```diff
@@ -92,6 +92,12 @@ impl OrthoFeeder {
         if !items.is_empty() {
             let new_orthos = db.upsert(items);
             workq.push_many(new_orthos);
+            
+            // If this is a real Queue (not MockQueue), ack the processed messages
+            let any_ref: &mut dyn std::any::Any = dbq;
```

Comment:
Don't use dyn. Make all queues fall under the same interface. All pops should return handles. Ack can be a no op for mock queue.